### PR TITLE
Add deadline + assignment standards

### PR DESCRIPTION
**Pull request by cfarm**
This adds standards for code review assignment + deadlines for merging PR's.

## Additions

- Recommends that a person creating a pull request assign a specific Github user to review the PR
- Recommends limiting reviewers to 1 person who will 👍  or 👎 
- Adds a 1 week deadline in which PRs can be self-merged if no feedback or comments are given.

## Notes

- We are testing this out on Capital Framework over the next few weeks, check back to see how it went.

### CODE REVIEW

**Comment by Scotchester** [comment]
How do we determine who to assign for review?

File: code-reviews.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ accomplish this, everyone should follow these tips.
 
 ### For code submitters
 
+- Assign one reviewer to your pull request using Github's interface.
+- Reminder reviewers to respond within 1 week, otherwise you are free to merge the PR if you receive no feedback.
```
Comment (Scotchester) [inline code-reviews.md]: Reminder ➡️  Remind

File: code-reviews.md [hunk 2]
```diff
@@ -59,6 +61,7 @@ accomplish this, everyone should follow these tips.
 
 ### For code reviewers
 
+- Review the code, make comments, or and/or merge within 1 week of being assigned a pull request.
```
Comment (Scotchester) [inline code-reviews.md]: or and/or ➡️  and/or

File: code-reviews.md (hunk 1 above)
Comment (KimberlyMunoz) [inline code-reviews.md]: Can we add "or have one :+1: sign off" to "receive no feedback" to make it clear that's the default guideline?

File: code-reviews.md [hunk 3]
```diff
@@ -45,6 +45,8 @@ accomplish this, everyone should follow these tips.
 
 ### For code submitters
 
+- Assign one reviewer to your pull request using Github's interface.
```
Comment (Scotchester) [inline code-reviews.md]: Trivial: Github ➡️  GitHub

**Comment by jimmynotjim** [comment]
> How do we determine who to assign for review?

The consensus on the call seemed to be a. those that have worked on it before and have existing knowledge, b. those that are on your team that the current work supports. There may be times when no one fits either of those roles, in which case it's probably a perfect situation to find someone willing to pair and knock it out quickly.

**Comment by Scotchester** [comment]
Makes sense! Should we include that guidance in the doc?

**Comment by jimmynotjim** [comment]
@Scotchester yeah, I think it's worth calling out, especially for when someone new joins the team.

**Comment by cfarm** [comment]
I remember from our CF meeting yesterday, it was suggested that we assign reviews to a team member from our project team. For those without that option, I think they can pick whoever they want.

**Comment by Scotchester** [comment]
:bell: @cfarm, could you address the review comments, please, so we can close this out?

**Comment by cfarm** [comment]
Fixes made. Please add the note about how to select a reviewer in another PR

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by Scotchester** [comment]
Why not just add it now? Do you anticipate an extended discussion about that language?

**Comment by KimberlyMunoz** [comment]
It's nice to keep the scope small and have it merged quickly.

**Comment by Scotchester** [comment]
The change is definitely within scope, and we're already at 20 days. What's another 5 minute commit?

**Comment by KimberlyMunoz** [comment]
The guidelines have a thing on this:

>If discussions turn too philosophical or academic, move the discussion to a new issue or offline. In the meantime, the author has the final say on the current issue or offline. In the meantime, the **author has the final say on the current implementation**.

**Comment by Scotchester** [comment]
That's fine, she can feel free to merge. I'm just asking questions.

I think we all need to get aligned on this topic, though. It makes no sense to me to consider that addition out of scope for this PR. Making another PR for something like this just drags things out just as much, only in a different way.

**Comment by ascott1** [comment]
I'm merging this. Let's add that other comment in a new PR to close discussion around this.

Thanks all of the changes and conversation.

**Pull request merged**